src/vs/workbench/contrib/notebook/browser/view/cellParts/cellOutput.ts

```diff
@@ -488,10 +488,18 @@ export class CellOutputContainer extends CellContentPart {
 	}
 
 	render() {
+		try {
+			this._doRender();
+		} finally {
+			// TODO@rebornix, this is probably not necessary at all as cell layout change would send the update request.
```

Comment:
is this still to be done in this PR?

Reply:
Will revisit this later (with our fork of list view)